### DISCUSSION TITLE
corret  fine-tune training logic

### DISCUSSION THREAD

**Discussion by lower01** [haotian-liu/LLaVA#557] [General]
Why do I need to download vicuna1.5? I saw that the modelpath in the fine-tuning script is vicuna1.5, and the output path is llava1.5. The related process is to first train the fine-tuning weights based on vicuna1.5 and the visual encoder, and then automatically integrate them into the llava1.5 model you have released?

I don’t have much experience in this area. My understanding is to directly download the llava1.5 you have released, fine-tune it based on llava1.5, mix the new fine-tuned weights with llava1.5, and get a new fine-tuned model.

I sincerely hope you can tell me the corret training logic.

**Comment by haotian-liu** [comment]
Hi, the finetuning instruction in README is on how to *reproduce* LLaVA-1.5. This means that we finetune Vicuna to a visual assistant.

Instructions and examples on finetuning LLaVA on some custom data will be released later.

Please let me know if there are other questions, thanks.
